packages/java/package.json

```diff
@@ -51,5 +51,6 @@
   ],
   "nyc": {
     "extends": "../../configs/nyc.json"
-  }
+  },
+  "jdt.ls.download.path" : "/jdtls/milestones/0.17.0/jdt-language-server-0.17.0-201804162308.tar.gz"
```

Comment:
just asking, how long will the milestones be preserved?

Reply:
The URL has milestones but these are  jdt.ls releases that are done bi-weekly. We do not remove them ever because other editors also retrieve them from here.